src/message/messageActionSheet.js

```diff
@@ -70,8 +72,15 @@ const copyToClipboard = async ({ _, auth, message }) => {
 copyToClipboard.title = 'Copy to clipboard';
 copyToClipboard.errorMessage = 'Failed to copy message to clipboard';
 
-const editMessage = async ({ message, dispatch }) => {
-  dispatch(startEditMessage(message.id, message.subject));
+const editMessage = async ({ message, dispatch, startEdit, auth }) => {
+  if (!message.isOutbox) {
```

Comment:
The case where `message.isOutbox` is true should be impossible -- it would represent a bug. (We only show this option for real, sent messages.) So the case where it's false should be treated as the main line of the logic, rather than put inside a condition.

A way to do that with the same semantics as this version would be an early return:
```js
  if (message.isOutbox) {
    return;
  }

  // …
```

But also as long as we have logic to detect this case -- which is here basically because we haven't expressed its impossibility in a way that the type-checker is able to prove for us -- we should have something to let us know if it starts impossibly happening:
```js
  if (message.isOutbox) {
    logging.warn('Attempted "Edit message" for outbox message');
    return;
  }

  // …
```

Another alternative would be to throw an exception. In this case just a warning is fine, because there's a pretty clear thing we can do to carry on -- just ignore the attempted action.